lib/common/config.go

```diff
@@ -46,8 +47,9 @@ func NewConfig(networkID []byte) Config {
 	p.BlockTime = 5 * time.Second
 	p.BlockTimeDelta = 1 * time.Second
 
-	p.TxsLimit = 1000
-	p.OpsLimit = 1000
+	p.TxsLimit = DefaultTransactionsInBallotLimit
+	p.OpsLimit = DefaultOperationsInTransactionLimit
+	p.OpsInBallotLimit = DefaultOperationsInBallotLimit
```

Comment:
OpsInBallotLimit should be always greater than OpsLimit
If not, some transaction may starve

Reply:
@kfangw Yes, right. I want to these opinion :)

Reply:
@kfangw This patch does not change the order of incoming transaction.